src/views/activity.py

```diff
@@ -0,0 +1,49 @@
+
+from flask import Blueprint, render_template, request, flash, redirect, url_for, make_response, g
+from src.models.activity import Activity
+from src.models.contact import Contact
+from utility.decor import login_required
+
+
+phonebook = Contact()
+activities = Activity()
+
+activity = Blueprint('activity', __name__)
+
+@activity.route('/<int:contact_id>', methods=["GET"])
+@login_required
+def get_all(contact_id):
+
+    contact_name = phonebook.get_by_id(contact_id)['name']
+
+    rows = activities.get_all(contact_id)
+
+    return render_template('activity.html',
+                           rows=rows,
+                           contact_name=contact_name,
+                           contact_id=contact_id,
+                           username=g.user['client_name'])
+
+
+@activity.route('/<int:contact_id>', methods=["POST"])
```

Comment:
This route should be `POST /contacts/:id/activities` as activities are a sub-resource for a contact.

Reply:
Right, sure